docs/src/main/paradox/jdbc.md

```diff
@@ -1,8 +1,246 @@
+
+@@@ note
+
+It is important that the `Handler` instance is not shared between several `Projection` instances,
+because then it would be invoked concurrently, which is not how it is intended to be used. Each `Projection`
+instance should use a new `Handler` instance.  
+
+@@@
+
+### Async handler
+
+The @apidoc[Handler] can be used with `JdbcProjection.atLeastOnceAsync` and 
+`JdbcProjection.groupedWithinAsync` if the handler is not storing the projection result in the database.
+The handler could @ref:[send to a Kafka topic](kafka.md#sending-to-kafka) or integrate with something else.
+
+There are several examples of such `Handler` in the @ref:[documentation for Cassandra Projections](cassandra.md#handler).
+Same type of handlers can be used with `JdbcProjection` instead of `CassandraProjection`.
+
+### Actor handler
+
+A good alternative for advanced state management is to implement the handler as an [actor](https://doc.akka.io/docs/akka/current/typed/actors.html),
+which is described in @ref:[Processing with Actor](actor.md).
+
+### Flow handler
+
+An Akka Streams `FlowWithContext` can be used instead of a handler for processing the envelopes,
+which is described in @ref:[Processing with Akka Streams](flow.md).
+
+### Handler lifecycle
+
+You can override the `start` and `stop` methods of the @apidoc[JdbcHandler] to implement initialization
+before first envelope is processed and resource cleanup when the projection is stopped.
+Those methods are also called when the `Projection` is restarted after failure.
+
+See also @ref:[error handling](error.md).
+
+## Schema
+
+The database schema for the offset storage table:
+
+```sql
+create table if not exists "AKKA_PROJECTION_OFFSET_STORE" (
+  "PROJECTION_NAME" VARCHAR(255) NOT NULL,
+  "PROJECTION_KEY" VARCHAR(255) NOT NULL,
+  "OFFSET" VARCHAR(255) NOT NULL,
+  "MANIFEST" VARCHAR(4) NOT NULL,
+  "MERGEABLE" BOOLEAN NOT NULL,
+  "LAST_UPDATED" TIMESTAMP(9) WITH TIME ZONE NOT NULL);
+
+create index "PROJECTION_NAME_INDEX" on "AKKA_PROJECTION_OFFSET_STORE" ("PROJECTION_NAME");
+
+alter table "AKKA_PROJECTION_OFFSET_STORE" 
+    add constraint "PK_PROJECTION_ID" primary key("PROJECTION_NAME","PROJECTION_KEY");
+```
+
+
+@@@ note
+The DDL may need to be adapted according to the specifics of your database brand.
+@@@
+
+## Offset types
+
+The supported offset types of the `JdbcProjection` are:
+
+* `akka.persistence.query.Offset` types from @ref:[events from Akka Persistence](eventsourced.md)
+* `MergeableOffset` that is used for @ref:[messages from Kafka](kafka.md)
+* `String`
+* `Int`
+* `Long`
+* Any other type that has a configured Akka Serializer is stored with base64 encoding of the serialized bytes.
+  For example the [Akka Persistence Spanner](https://doc.akka.io/docs/akka-persistence-spanner/current/) offset
+  is supported in this way.
+
+## Configuration
+
+Make your edits/overrides in your application.conf.
+
+The reference configuration file with the default values:
```

Comment:
Maybe add a note here that dialect and fixed-pool-size must be configured by the application.
Would be good to highlight that somewhere in the beginning of the doc also. There is a clear runtime error message but good to make it clear upfront that the config is needed.